lib/burials/monitor.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module Burials
+  ##
+  # Monitor functions for Rails logging and StatsD
+  #
+  class Monitor
```

Comment:
income-and-assets has this separated out to a claims monitor and submission monitor (not part of this ticket).  not sure if you want to follow suit now, or wait until the move to the module and/or the abstraction of the monitoring functions

Reply:
@wayne-weibel I think its okay to wait until we move into it's own module, less files to port over